package.json

```diff
@@ -49,7 +49,7 @@
     "ink-docstrap": "^1.1.4",
     "jsdoc": "^3.6.3",
     "jshint": "^2.10.2",
-    "keycloak-admin-client": "bucharest-gold/keycloak-admin-client",
+    "keycloak-admin-client": "abstractj/keycloak-admin-client",
```

Comment:
Why not use the latest version of [`keycloak-admin-client`](https://www.npmjs.com/package/keycloak-admin-client) that was published on NPM? Does this repo contain changes that are missing there?

Reply:
@jonkoops because I missed that on npm :) Thanks for that, PR updated. If all the checks pass, I will merge it so we can discuss more about the migration next year, and I don't have to bother you again.